VERSIONS.md

```diff
@@ -0,0 +1,19 @@
+# Versions
+
+The following table describes the versions of this project:
+
+| Version          | Supported                           | Minimum Node.js | Branch | Npm Tag            | Docker Images             |
+| ---------------- | ----------------------------------- | --------------- | ------ | ------------------ | ------------------------- |
+| 4.x              | :x: (deprecated)                    | 10              | 4.x    | latest-4           | 4, 4.x, 4.x.x, 4.x-next   |
+| 5.x previous     | :white_check_mark: (until Nov 2024) | 14              | 5.x    | latest-5           | 5, 5.x, 5.x.x, 5.x-next   |
+| _6.x current_    | :white_check_mark:                  | _18_            | _6.x_  | _latest-6, latest_ | _6, 6.x, 6.x.x, 6.x-next_ |
+| 7.x next         | :x:                                 | 18              | 7.x    | next-7             | 7.x-next                  |
+| 8.x experimental | :x:                                 | 18              | master | next-8             | nightly-master            |
```

Comment:
I just noticed, the README also need an small update to use `next-8` instead `next`